documentation/specs/src/interoperability/ethereum-bridge/bootstrapping.md

```diff
@@ -1,117 +1,115 @@
+Whether the bridge is started at genesis or post-genesis (through a governance
+proposal), validator set update proofs are always available to bootstrap the
+bridge and maintain its liveness.
 
-The governance proposal can be freeform and simply indicate what the value of
-`h` should be. Validators should then configure their nodes to halt at this
-height. The `grace_epoch` is arbitrary as there is no code to be executed as
-part of the proposal, instead validators must take action manually as soon as
-the proposal passes. The block height `h` must be in an epoch that is strictly
-greater than `voting_end_epoch`.
-
-### Value for launch height `h`
-
-The active validator set at the launch height chosen for starting the Ethereum
-bridge will have the extra responsibility of restarting the chain if they
-consider the deployed smart contracts as valid. For this reason, the validator
-set at this height must be known in advance of the governance proposal
-resolving, and a channel set up for offchain communication and co-ordination of
-the chain restart. In practise, this means the governance proposal to launch the
-chain should commit to doing so within an epoch of passing, so that the
-validator set is definitely known in advance.
-
-### Deployer
-
-Once the smart contracts are fully deployed, only the active validator set for
-block height `h` should have control of the contracts so in theory anyone could
-do the Ethereum bridge smart contract deployment.
-
-### Backing out of Ethereum bridge launch
+## Overview
 
-If for some reason the validity of the smart contract deployment cannot be
-agreed upon by the validators who will responsible for restarting Namada, it
-must remain possible to restart the chain with the Ethereum bridge still not
-enabled.
+To bootstrap the Ethereum bridge, there are six governance parameters which
+must be written to storage:
+
+- `eth_bridge_min_confirmations` - The minimum number of block confirmations
+  on Ethereum required for any given event to be voted on by Namada validators.
+- `eth_bridge_bridge_address` - The address of the `Bridge` contract, used to
+  perform transfers in either direction (Namada <> Ethereum).
+- `eth_bridge_bridge_version` - The version of the `Bridge` contract, starting
+  from 1.
+- `eth_bridge_governance_address` - The address of the `Governance` contract,
+  used to perform administrative tasks, such as updating validator sets in
+  Ethereum.
+- `eth_bridge_governance_version` - The version of the `Governance` contract,
+  starting from 1.
+- `eth_bridge_wnam_address` - The address of the deployment of the native
+  ERC20 token, representing NAM in Ethereum.
+
+Here is an overview of the steps to follow to enable the Ethereum bridge through a
+governance proposal for a given Namada chain:
+
+1. The [smart contracts](./ethereum_smart_contracts.md) of the Ethereum bridge
+   should be deployed a priori, with an initial validator set and other parameters,
```

Comment:
```suggestion
   should be deployed beforehand, with an initial validator set and other parameters,
```

I get what you mean :smile:, and from Namada's perspective it is _a priori_, but this might be confusing for some readers).